team-fingerprints-web/src/types/models.d.ts

```diff
@@ -124,3 +125,26 @@ export type AdditionalData = {
   id: string;
   name: string;
 };
+
+export type Filter = {
+  _id: string;
+  filterPath: string;
+  name: string;
+  values: FilterValue[];
+};
+
+export type FilterValue = {
+  _id: string;
+  value: string;
+};
+
+export type FormData = {
+  [key: string]: string;
+};
+
+export type ProfileSelector = {
```

Comment:
I think there's no need to have it in a different file. BTW I also think it's nice to have "props" word in type name to explicitly show it's a props type

```suggestion
export type ProfileSelectorProps = {
```